### ISSUE TITLE
someone locked my packages ;(

### ISSUE THREAD

**Issue by coolo** [openSUSE/open-build-service#1256]
```
#~>osc rdelete openSUSE:Factory kernel-desktop -f -mdrop
Server returned an error: HTTP Error 400: Bad Request
Package is used by following packages as devel package: home:vmwareideas:kernel-bfs/kernel-desktop-bfs
```

There is no way for me to delete this package ;(

**Comment by adrianschroeter** [comment]
yep, regression, could be there since refactoring of some coolo guy :) fixed in pull request now.

**Comment by coolo** [comment]
what pull request?

I still can't delete kernel-desktop

**Comment by adrianschroeter** [comment]
fixed with aca1ed8b0e0170f2bcbd453cec00e978a9e98779

**Comment by coolo** [comment]
it's gone! :)
